explorer/models.py

```diff
@@ -112,6 +112,22 @@ class Meta:
         ordering = ['-run_at']
 
 
+class QueryChangeLog(models.Model):
+
+    old_sql = models.TextField(null=True, blank=True)
+    new_sql = models.TextField(null=True, blank=True)
+    query = models.ForeignKey(Query, null=True, blank=True, on_delete=models.SET_NULL)
+    run_by_user = models.ForeignKey(settings.AUTH_USER_MODEL, null=True, blank=True)
```

Comment:
does this represent which user changed the SQL?

Reply:
Yes, it stores which user changed or modified the SQL.